### DISCUSSION TITLE
eBPF-for-windows demo with bind() hook

### DISCUSSION THREAD

**Discussion by poornagmsft** [microsoft/ebpf-for-windows#706] [Show and tell]
This is a demo of the ebpf-for-windows platform with a hook based on the Windows Filtering Platform's resource assignment layer that can be used to track bind() requests from applications.
The code for this is available at tests/sample/bindmonitor.c

https://user-images.githubusercontent.com/62666514/147957561-68e71793-7c18-4f4c-8880-4feffc1e8869.mp4

**Comment by dthaler** [comment]
Thanks Poorna, and great demo by Alan.   Maybe we could add a tutorial page on how anyone can reproduce this demo on their own machine?

**Reply by poornagmsft** [reply]
Yes, we will follow up. That is a great suggestion.

**Comment by Alan-Jowett** [comment]
We should also figure out how to add subtitles for this video to make it easier to follow.

**Reply by nibanks** [reply]
My recommendation is to upload to youtube and link to there. Youtube automatically does the subtitles.
